wire-runtime/src/commonMain/kotlin/com/squareup/wire/Message.kt

```diff
@@ -30,7 +30,7 @@ expect abstract class Message<M : Message<M, B>, B : Message.Builder<M, B>> prot
    * Returns a byte string containing the proto encoding of this message's unknown fields. Returns
    * an empty byte string if this message has no unknown fields.
    */
-  val unknownFields: ByteString
+  fun unknownFields(): ByteString
```

Comment:
This shouldn't have to change to support the behavior.

Reply:
Lets try something completely different:
https://github.com/square/wire/pull/1160/files